Footer changes (icons fixed, more responsive)

wgbowley opened the pull request:
We do not have the copyright to 'Skill Tree Project' yet. Didn't know that, I had to do a pull request

wgbowley committed:
Remove all rights reserved 

We do not have the copyright to 'Skill Tree Project' yet.

faderzz committed:
mobile responsive icons - footer

smaller icons but larger space between them on mobile devices.

faderzz committed:
desktop footer icons (responsive)

slightly decreased the size of the icons on the footer

faderzz committed:
icons fixed

icons were not aligned properly due to the aspect ratio of some images not being 1:1 - now changed, no reduction in image quality as the actual icon has not been affected

faderzz reviewed (dismissed):
PR fixes many footer issues and makes the footer much more responsive - Improves user experience in general

wgbowley committed:
Box shadow for Contribute-images

wgbowley committed:
Removing logs

Removing console log, we no longer need to print the contents and contents max height to the console because we're finished.
